### ISSUE TITLE
Write an introduction/about text for the Home page

### ISSUE THREAD

**Issue by taneltm** [codedtogether/web#6]
This should be done after #5 

The current intro text is written hastily and should be improved.

**Comment by abcdan** [comment]
We've changed the homepage, can I close the issue?

**Comment by anishanne** [comment]
```
Welcome to the official website for the Code Together project, it's an amazing place to learn from other developers and teach people things. With over 1500 people and new people joining every day you'll always find someone who can help you out.
```

I think the "With over 1500 people" should be pulled directly via a Discord API as well as a text size increase would be a good idea.
![image](https://user-images.githubusercontent.com/22670462/67726327-cc1a7f80-f9bb-11e9-8e0c-1558ff56c8f6.png)

**Comment by SpyHoodle** [comment]
I’ve done a more in depth about section on the site, but I suggest we keep this open until the user count @anishanne suggested is done if you want to do it or not.

**Comment by SpyHoodle** [comment]
# Update:
@A-da-m has done the automated user count and will PR soon sometime later.

**Comment by A-da-m** [comment]
Closed in PR #30
